Magic/src/main/java/com/elmakers/mine/bukkit/action/builtin/ModifyAttributeAction.java

```diff
@@ -48,6 +48,14 @@ public void prepare(CastContext context, ConfigurationSection parameters) {
         for (String key : section.getKeys(false)) {
             attributes.put(key, section.getDouble(key, 0));
         }
+
+        for (String key : parameters.getKeys(false)) {
+            if (key.startsWith("attribute_")) {
+                String str = key.replace("attribute_", "");
```

Comment:
I have actually been trying to discourage this type of parametrization (such as in PotionEffectAction, I'd prefer add_effects versus effect_slow, etc). So I'm not sure about this part, sometimes fewer options can be better, or certainly less confusing.

Reply:
Will be removing this in the next commit.